### ISSUE TITLE
PBR addon't wont work

### ISSUE THREAD

**Issue by vchslvsvs** [skurtyyskirts/GmodRTX#5]
Using 0.4 alpha. The only enhancement addon working - Water. Any other (Mirror, default PBR, for specific map) does not works

**Comment by vchslvsvs** [comment]
![image](https://github.com/user-attachments/assets/a2e881e7-1af1-463c-b25d-55d3d3c488f2)
![image](https://github.com/user-attachments/assets/98bf2df8-1bd7-40c6-a9b8-e4381740c79b)

**Comment by skurtyyskirts** [comment]
Thanks for the report! I'll see why the hash update to gmod.exe didn't replace the mirror, might have to manually tweak the mod.usda file. If that doesn't work I'll just make a new mod file from scratch for it, will report back.

**Comment by skurtyyskirts** [comment]
It looks like the flickering water was the result of a recent PR from the RTX-Fixes addon, this has since been corrected. On top of this, CR was kind enough to develop a lua script that I'll be implementing to the installation that fixes the milky water that appears in Episode 2, which was never covered with the water mod.

The mirror has also been corrected, the glass is also performing as expected. As far as the PBR for gm_construct_rtx, the GmodRTX installer doesn't come with a PBRpack. But, I have plans on updating the Gm_Construct_RTX PBR map addon over on ModDB.

**Fixes will ship with the next GmodRTX update.**

I really appreciate you reporting this, as it only makes the mod better for newcomers.

**Comment by vchslvsvs** [comment]
i was noticed, that PBR addons from ModDB for specific maps (tested on CS:GO safehouse and 1 more) are partially works. They are not replacing textures, but turning  "Assets enhanchment" on turns off all light sources on the map.

**Comment by skurtyyskirts** [comment]
Yeah it's very unfortunate - Garrysmod recently had an update where they removed the hl2.exe from their installation process. Before, garrysmod would launch, but in task manager, hl2.exe would be running. Because of this, all of the mods I've made have been built on hl2.exe hashes. Now that Garrysmod has shifted to using their own .exe, the hashes have all updated and changed. Because of this, all of my mods are basically broken. I was able to make a python script to to help change the hashes, but only certain projects/maps had the information I needed.

But I have updated these so far/added,
https://www.moddb.com/mods/destroyable-city-rtx
https://www.moddb.com/mods/the-flying-hotel-rtx
https://www.moddb.com/mods/gm-pool-complex-v2-rtx/downloads/gm-pool-complex-v2-version-2

Those 3 maps will work with the current gmodrtx :) 

Tl;DR Gmod had an update that ruined all the old mods lol
